doc/manual/source/manual-page.rst

```diff
@@ -1378,6 +1378,16 @@ The service only supports GET requests with the following paths:
       provided session and serial. If *reset* is *true*, the *withdrawn*
       member is not present.
 
+/json-delta/notify, /json-delta/notify?session=session&serial=serial
+      Waits with a response until a new update is available and then returns
+      a JSON object with two members *session* and *serial* which contain the
+      session ID and serial number of the updated data set.
+
+      If the *session* and *serial* query parameters are provided, the JSON
+      object is returned immediately if the session ID or serial number of
+      the current data set differ from the provided values and only waits
+      for an update if they are identical.
+
 In addition, the current set of VRPs is available for each output format at a
```

Comment:
> If the session and serial query parameters are provided,
What happens if I don't provide them? Also, don't have man pages have a syntax convention to indicate that query parms are optional?

Reply:
Maybe also indicate where the user could get the current session and serial info from? Or is that obvious from the workflow of using routinator?

Reply:
Maybe it would be better – and more consistent – to also return immediately if session/serial are missing? I.e., only wait if the current session/serial are provided?

Reply:
Ack. Change incoming.